### ISSUE TITLE
Can manual intelligent shooting be supported? In dart, users sometimes want to shoot by themselves.

### ISSUE THREAD

**Issue by YaYaG** [eliasteeny/flutter_document_scanner#11]
Can manual intelligent shooting be supported? In dart, users sometimes want to shoot by themselves.

**Comment by eliasteeny** [comment]
Yeah manual shooting can be supported. Actually this feature is available but is broken for now, but I have been very busy lately to fix it.
